lxd/db/cluster/instance_profiles.go

```diff
@@ -1,3 +1,5 @@
+//go:build linux && cgo && !agent
```

Comment:
Does this mean that lxd-agent is importing lxd/db/cluster package? If so this seems like a bug?